examples/pyfunc/train_custom_model.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from mlflow.tracking import MlflowClient
+from sklearn.linear_model import LogisticRegression
+
+import mlflow
+import mlflow.sklearn
+
+
+class CustomPredict(mlflow.pyfunc.PythonModel):
+    def __init__(self) -> None:
+        """Custom pyfunc class used to create customized mlflow models"""
```

Comment:
Do we need this empty constructor?

Reply:
```suggestion
    """Custom pyfunc class used to create customized mlflow models"""
```

Can we use class docstring?